cypari2/closure.pyx

```diff
@@ -133,39 +135,40 @@ cpdef Gen objtoclosure(f):
         be interrupted. Therefore, it is advised to use this only for
         simple functions which do not take a long time.
 
-    EXAMPLES::
-
-        sage: from sage.libs.cypari2.closure import objtoclosure
-        sage: mul = objtoclosure(lambda i,j: i*j)
-        sage: mul
-        (v1,v2,v3,v4,v5)->call_python(v1,v2,v3,v4,v5,...)
-        sage: mul.type()
-        't_CLOSURE'
-        sage: mul(6,9)
-        54
-        sage: def printme(x):
-        ....:     print(x)
-        sage: objtoclosure(printme)('matid(2)')
-        [1, 0; 0, 1]
-
-    Test various kinds of errors::
-
-        sage: mul(4)
-        Traceback (most recent call last):
-        ...
-        TypeError: <lambda>() takes exactly 2 arguments (1 given)
-        sage: mul(None, None)
-        Traceback (most recent call last):
-        ...
-        ValueError: Cannot convert None to pari
-        sage: mul(*range(100))
-        Traceback (most recent call last):
-        ...
-        PariError: call_python: too many parameters in user-defined function call
-        sage: mul([1], [2])
-        Traceback (most recent call last):
-        ...
-        PariError: call_python: forbidden multiplication t_VEC (1 elts) * t_VEC (1 elts)
+    Examples:
+
+    >>> from cypari2.closure import objtoclosure
+    >>> def pymul(i,j): return i*j
```

Comment:
?? What do you mean? I'm using Python 3.6, and I see no problem with the old test

```
>>> objtoclosure(lambda i,j: i*j)
(v1,v2,v3,v4,v5)->call_python(v1,v2,v3,v4,v5,140180350752424)
```